flow/layers/transform_layer.cc

```diff
@@ -56,8 +56,12 @@ void TransformLayer::UpdateScene(SceneUpdateContext& context) {
   TRACE_EVENT0("flutter", "TransformLayer::UpdateScene");
   FML_DCHECK(needs_system_composite());
 
-  SceneUpdateContext::Transform transform(context, transform_);
-  UpdateSceneChildren(context);
+  if (!transform_.isIdentity()) {
```

Comment:
Nice!  this should cut down on the node count a good bit until we can roll out Embedder API

Reply:
Yep, it got rid of a few nodes. The changes to OpacityLayer removed even more.